vignettes/dashboard.Rmd

```diff
@@ -22,6 +22,8 @@ The data in this table stems from our querying ftp://cran.r-project.org/incoming
 We update it every hour. [See below](#cran-review-workflow) for a description of each 
 folder meaning.
 
+![](https://github.com/r-hub/cransays/workflows/Render-dashboard/badge.svg)
```

Comment:
```suggestion
![Dashboard status](https://github.com/r-hub/cransays/workflows/Render-dashboard/badge.svg)
```

Reply:
I tried this at first but it looks like it also add a caption and not just an alt text. Additionally, for people using screen readers, I don't think it's super useful to know that there is a status button if they don't know the actual status. I'm not sure what is the right choice here.

Reply:
In that case should it be added with HTML that adds an aria hidden attribute or something similar? I do not know what the right answer is.